Source/WebCore/accessibility/AXTextMarker.cpp

```diff
@@ -858,6 +939,59 @@ AXTextMarkerRange AXTextMarker::lineRange(LineRangeType type) const
     RELEASE_ASSERT_NOT_REACHED();
 }
 
+AXTextMarkerRange AXTextMarker::wordRange(WordRangeType type) const
+{
+    if (!isValid())
+        return { { }, { } };
+
+    AXTextMarker startMarker, endMarker;
+
+    if (type == WordRangeType::Right) {
+        startMarker = nextWordStart();
+        endMarker = startMarker.nextWordEnd();
+
+        if (startMarker == *this) {
+            auto resultRange = AXTextMarkerRange { startMarker, endMarker };
+            auto resultRangeString = resultRange.toString();
+            // A newline should not be returned as the right word (if we are at the end of a paragraph)
+            if (resultRangeString.length() && resultRangeString[0] == '\n')
```

Comment:
Is it possible to simplify this like so:

```cpp
auto rangeString = AXTextMarkerRange { startMarker, endMarker }.toString();
// A newline should not be returned as the right word (if we are at the end of a paragraph)
if (rangeString() && rangeString[0] == '\n')
```

Reply:
Yep, I'll change this.